test/verify/check-packagekit

```diff
@@ -392,7 +396,7 @@ class TestUpdates(MachineCase):
 
         # make sure we have enough time to crash PK
         self.createPackage("slow", "1", "1", install=True)
-        self.createPackage("slow", "1", "2",  postinst='sleep 10')
+        self.createPackage("slow", "1", "2",  postinst="while [ ! -e /tmp/finish-pk ]; do sleep 1; done")
```

Comment:
I was wondering about that, and consistency. I'll use `sleep infinity` and add a comment